snc.sh

```diff
@@ -8,7 +8,7 @@ JQ=${JQ:-jq}
 OC=${OC:-oc}
 YQ=${YQ:-yq}
 OPENSHIFT_INSTALL=${OPENSHIFT_INSTALL:-./openshift-install}
-OPENSHIFT_RELEASE_VERSION=$(git describe --abbrev=0 --tags 2>/dev/null) || OPENSHIFT_RELEASE_VERSION=
+OPENSHIFT_RELEASE_VERSION=$(git describe --exact-match --tags HEAD 2>/dev/null) || OPENSHIFT_RELEASE_VERSION=
```

Comment:
Sure, though I'm worried this might need to be more sophisticated, as we want totest 4.1.latest, and probably y4.2.latest. Maybe this should be handled through installer build parameters rather than with version overrides here

Reply:
I just mean that I foresee more work needed in that area, this particular commit is fine with me though, and the patch series too.